src/sortTemplate.ts

```diff
@@ -50,81 +50,89 @@ export async function sortTemplate(template: DeploymentTemplate | undefined, sec
     ext.outputChannel.appendLine("Sorting template");
     switch (sectionType) {
         case TemplateSectionType.Functions:
-            await sortFunctions(template, textEditor);
+            await showSortingResultMessage(() => sortFunctions(template, textEditor), "Functions");
             break;
         case TemplateSectionType.Outputs:
-            await sortOutputs(template, textEditor);
+            await showSortingResultMessage(() => sortOutputs(template, textEditor), "Outputs");
             break;
         case TemplateSectionType.Parameters:
-            await sortParameters(template, textEditor);
+            await showSortingResultMessage(() => sortParameters(template, textEditor), "Parameters");
             break;
         case TemplateSectionType.Resources:
-            await sortResources(template, textEditor);
+            await showSortingResultMessage(() => sortResources(template, textEditor), "Resources");
             break;
         case TemplateSectionType.Variables:
-            await sortVariables(template, textEditor);
+            await showSortingResultMessage(() => sortVariables(template, textEditor), "Variables");
             break;
         case TemplateSectionType.TopLevel:
-            await sortTopLevel(template, textEditor);
+            await showSortingResultMessage(() => sortTopLevel(template, textEditor), "Top level items");
             break;
         default:
             assertNever(sectionType);
-
     }
-    vscode.window.showInformationMessage("Done sorting template!");
 }
 
-async function sortOutputs(template: DeploymentTemplate, textEditor: vscode.TextEditor): Promise<void> {
+async function showSortingResultMessage(sortAction: () => Promise<boolean>, part: string): Promise<void> {
```

Comment:
Please document the return value

Reply:
Fixed.

Reply:
Thanks. The `@returns True if order was changed when sorting, otherwise false` part was what I was most concerned with, since boolean is rarely self-evident.  My personal philosophy on comments - unless it's an API for external consumption, I stick mostly to non-obvious stuff.